internal/hcops/load_balancer.go

```diff
@@ -1069,6 +1068,20 @@ func (b *hclbServiceOptsBuilder) extractHealthCheck() {
 		return nil
 	})
 
+	b.do(func() error {
+		port, err := annotation.LBSvcHealthCheckTLSPort.IntFromService(b.Service)
```

Comment:
You could use `IntsFromService()` to get an `[]int` back for allowing multiple ports.